src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -235,50 +522,83 @@ ${SDK_PATH}/tools/${ARCH}/ffx product-bundle get workstation_eng.qemu-${ARCH}
 ${SDK_PATH}/tools/${ARCH}/ffx emu start workstation_eng.qemu-${ARCH} --headless
 ```
 
-Once the emulator is running, start a package repository server to serve our
-package to the emulator:
+### Watching emulator logs
 
+Once the emulator is running, open a separate terminal to watch the emulator logs:
+
+**In separate terminal**
 ```sh
-${SDK_PATH}/tools/${ARCH}/ffx repository server start
+${SDK_PATH}/tools/${ARCH}/ffx log \
+    --since now
 ```
 
-Once the repository server is up and running, register our repository:
+### Serving a Fuchsia package
+
+Now, start a package repository server to serve our
+package to the emulator:
 
 ```sh
-${SDK_PATH}/tools/${ARCH}/ffx target repository register --repository hello-fuchsia
+${SDK_PATH}/tools/${ARCH}/ffx repository server start
 ```
 
-And watch the logs from the emulator in a separate terminal:
+Once the repository server is up and running, register our repository:
```

Comment:
```suggestion
Once the repository server is up and running, register it with the target Fuchsia system running in the emulator:
```

I had to do a double take because I thought this was the same kind of repo registration as we did above with the `add-from-pm` command.